spec/grape/validations/validators/coerce_spec.rb

```diff
@@ -397,6 +397,10 @@ def self.parsed?(value)
         post '/upload', file: 'not a file'
         expect(last_response.status).to eq(400)
         expect(last_response.body).to eq('file is invalid')
+
+        post '/upload', file: { filename: 'fake file', tempfile: '/etc/passwd' }
+        expect(last_response.status).to eq(400)
+        expect(last_response.body).to eq('file is invalid')
```

Comment:
I know we're just copy-pasting, but for next time each of these should be a separate spec with a `before` block that sets up the `subject`.